### PR TITLE
映射文档 No. 165-184

### PR DESCRIPTION
**Pull request by co63oc**
https://github.com/PaddlePaddle/PaConvert/issues/112 

165 torch.distributions.exp_family.ExponentialFamily 已有文档修改
166 torch.distributions.bernoulli.Bernoulli
167 torch.distributions.binomial.Binomial  不支持API
168 torch.distributions.categorical.Categorical 已有文档
169 torch.distributions.cauchy.Cauchy 
170 torch.distributions.continuous_bernoulli.ContinuousBernoulli  不支持API
171 torch.distributions.dirichlet.Dirichlet 已有文档修改
172 torch.distributions.exponential.Exponential  不支持API
173 torch.distributions.geometric.Geometric  
174 torch.distributions.gumbel.Gumbel  
175 torch.distributions.independent.Independent
176 torch.distributions.laplace.Laplace
177 torch.distributions.log_normal.LogNormal
178 torch.distributions.multinomial.Multinomial 已有文档修改
179 torch.distributions.normal.Normal 已有文档修改
180 torch.distributions.one_hot_categorical.OneHotCategorical  不支持API
181 torch.distributions.transformed_distribution.TransformedDistribution 已有文档
182 torch.distributions.uniform.Uniform 已有文档修改
183 torch.distributions.transforms.Transform 已有文档
184 torch.distributions.transforms.ComposeTransform 已有文档

### CODE REVIEW

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.bernoulli.Bernoulli.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+```
+
+Pytorch 相比 Paddle 支持更多其他参数，具体如下：
+
+### 参数映射
+
+| PyTorch       | PaddlePaddle | 备注                                                                    |
+| ------------- | ------------ | ----------------------------------------------------------------------- |
+| probs         | probs        | 伯努利分布的概率输入。                                                  |
+| logits        | logits       | 采样 1 的 log-odds。                                                    |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.bernoulli.Bernoulli.md]: 这个参数paddle没有吧

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md [hunk 2]
```diff
@@ -737,6 +737,12 @@
...
 | 2    | [torch.distributions.kl.register_kl](https://pytorch.org/docs/stable/distributions.html?highlight=register_kl#torch.distributions.kl.register_kl) | [paddle.distribution.register_kl](https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/distribution/register_kl_cn.html) | 功能一致，仅参数名不一致， [差异对比](https://github.com/PaddlePaddle/docs/tree/develop/docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.kl.register_kl.md)         |
+| 3    | [torch.distributions.binomial.Binomial](https://pytorch.org/docs/stable/distributions.html#torch.distributions.binomial.Binomial) |  | 功能缺失        |
+| 4    | [torch.distributions.continuous_bernoulli.ContinuousBernoulli](https://pytorch.org/docs/stable/distributions.html#torch.distributions.continuous_bernoulli.ContinuousBernoulli) |  | 功能缺失        |
+| 5    | [torch.distributions.exponential.Exponential](https://pytorch.org/docs/stable/distributions.html#torch.distributions.exponential.Exponential) |  | 功能缺失        |
```
Comment (zhwesky2010) [inline docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:742]: 要以最新的paddle为准，这两个都已经有：
infoflow 2023-07-24 12-11-04

File: docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.bernoulli.Bernoulli.md (hunk 1 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/api_difference/distributions/torch.distributions.bernoulli.Bernoulli.md]: 已修改

File: docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md (hunk 2 above)
Comment (co63oc) [reply docs/guides/model_convert/convert_from_pytorch/pytorch_api_mapping_cn.md:742]: 已修改添加Gumbel, Geometric

**Review by zhwesky2010 — APPROVED** [review]
LGTM
